operator/v1/types_console.go

```diff
@@ -40,9 +44,14 @@ type ConsoleCustomization struct {
 	// Invalid value will prevent a console rollout.
 	DocumentationBaseURL string `json:"documentationBaseURL,omitempty"`
 	//customProductName is the customizable product name that can be set for the web console
+	// +optional
 	CustomProductName string `json:"customProductName"`
-	//customLogoFile is the name of the file that contains a customized product logo that can be set for the web console
-	CustomLogoFile string `json:"customLogoFile"`
+	//customLogoFile references a ConfigMap containing a custom product logo that can be set for the web console
+	// You only have to set this if you have your own custom product logo you wish to show on the console.
+	// The ConfigMap must exist in the openshift-config namespace and contain the following required fields:
+	// - ConfigMap.Data["filepath"] - logo location.
+	// +optional
+	CustomLogoFile v1.ConfigMapNameReference `json:"customLogoFile"`
```

Comment:
`CustomLogoFile` AND the custom product name/brand should both be set, if either is used, correct?  It would be strange to set a logo file & not a product name, etc.  I assume we will use the custom name as the key & will ignore logo file if name is not set, but it may be good to doc this in the comment.

Reply:
Again I am open to suggestions. I looked through the console code in https://github.com/openshift/console/pull/1391 and it appears the logic allows each item to be set independently.  I definitely could see requiring both, but I am not sure the use-cases.  Perhaps someone does want a custom name like "Bob's Awesome Openshift Cluster" with the standard RedHat Logo, or perhaps someone wants the "OKD" brand but with their own super cool logo graphic.
I am fine with leaving them separate items with no dependency because the worse thing that happens is the admin gets a custom name without his logo, in which he then needs to add the logo file, or vice versa.  Either way, no real harm.  Let me know what you think @benjaminapetersen

Reply:
> CustomLogoFile AND the custom product name/brand should both be set, if either is used, correct?

I don't see a reason to force them both to be set. The console handles it either way.